docs/descriptors/xb1sx.md

```diff
@@ -92,16 +92,16 @@ Parsed descriptor:
 0x81, 0x03,        //   Input (Const,Var,Abs,No Wrap,Linear,Preferred State,No Null Position)
 0x05, 0x09,        //   Usage Page (Button)
 0x19, 0x01,        //   Usage Minimum (0x01)
-0x29, 0x0B,        //   Usage Maximum (0x0B)
+0x29, 0x0F,        //   Usage Maximum (0x0F)
 0x15, 0x00,        //   Logical Minimum (0)
 0x25, 0x01,        //   Logical Maximum (1)
 0x75, 0x01,        //   Report Size (1)
-0x95, 0x0B,        //   Report Count (11)
```

Comment:
Ah okay, so the original rdesc has 15 buttons and MS just uses one of the 15 bits to report the button.